configuration.py

```diff
@@ -12,7 +12,15 @@ class Configuration(Enum):
 
     # ----------  BOT CONFIGURATION  ------------ #
     NUMBER_OF_SENTENCES_IN_RESPONSE = 2
-    REQUESTS_IN_ROW_THRESH = 5
+    REQUESTS_IN_ROW_THRESH = 3
+    POP_QUEST_BOT_CONF_THRESH = 0.65
+    POP_QUEST_BOT_CONST_CONF = 0.98
+    DEFAULT_CONF = 0.0
+    MAX_CONF = 1.0
+
+    # OTHER POPULAR BOT UTILS
+    SINGLE_NUM_KEYWORDS = ['ile', 'ilu' ]
+    COMP_NUM_KEYWORDS = ['jak', 'wiele', 'dużo', 'wielu']
```

Comment:
I am not sure if it is the right place to put it, I will place it in statements collection in db

Reply:
precious remark 
#my#precious...#remark